src/feed.py

```diff
@@ -0,0 +1,64 @@
+from selenium import webdriver
+from selenium.webdriver.common import keys
+from selenium.webdriver.firefox.options import Options as Firefox_Options
+from selenium.webdriver.common.keys import Keys
+from time import sleep
+from xpath_library import XPATHS
+import selenium_utils as su
+from selenium_utils import log_error, search_by_xpath, get_attribute, current_url
+
+class Post:
+    def __init__(self, data):
+        self.data = data
```

Comment:
This data is a dictionary which you presume the structure of... this is strong cuppling